### ISSUE TITLE
Teaching bubble close icon does not have a tooltip

### ISSUE THREAD

**Issue by caiomontei** [OfficeDev/office-ui-fabric-react#10010]
## Codepen demonstrating the issue
https://codepen.io/anon/pen/GVErJr

## Actual behavior:
Tabbing or hovering to the close icon does not show any tooltip.
The close button also does not contain an aria label.
## Expected behavior:
A tooltip should appear indicating that the icon is used for closing the callout.
The button should have an aria label so it is announced by screen readers.

## Documentation describing expected behavior
https://www.w3.org/WAI/WCAG21/Understanding/sensory-characteristics.html
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

**Comment by marygans** [comment]
Hi @caiomontei, Thanks foe submitting this issue! I have confirmed that this is an issue and we will work on getting it fixed!

**Comment by marygans** [comment]
Hi @caiomontei `closeButtonAriaLabel="Close"` needs to be added for the tooltip and aria label to be shown/read as you can see in the example [here](https://codepen.io/gwiz0210/pen/eqradj). I will make sure the `Teachingbubble` examples include that. 
Thanks :)

**Comment by caiomontei** [comment]
hi @marygans,

Thanks for the info. Unfortunately, this just solves the problem partially, since the tooltip just appears on mouse hover. If the user is using keyboard and focus on the icon, nothing appears.

Thanks

**Comment by marygans** [comment]
> hi @marygans,
> 
> Thanks for the info. Unfortunately, this just solves the problem partially, since the tooltip just appears on mouse hover. If the user is using keyboard and focus on the icon, nothing appears.
> 
> Thanks

I see. In Edge, tooltip on focus is a default behavior, but not for other browsers and thats intended behavior for this component. 

Can I suggest using the [tooltip ](https://developer.microsoft.com/en-us/fabric#/controls/web/tooltip) component for that specific behavior? which is a more intentional activation of a tooltip on focus.

**Comment by caiomontei** [comment]
hi @marygans,

What do you mean by using the Tooltip component? Completely replace the teaching bubble with a Tooltip, or somehow use the tooltip component inside the teaching button for the close icon. 

If it is the latter, I could not find any callbacks where I can provide my custom close icon implementation.

thanks
